### DISCUSSION TITLE
What is the difference between apiserver and binding-apiserver？

### DISCUSSION THREAD

**Discussion by DAJINZI01** [clusterpedia-io/clusterpedia#703] [Q&A] [answered]
What is the difference between apiserver and binding-apiserver？Is binding-apiserver just a apiserver with a synchromanager. Which one  should be use when deploy the clusterpedia

**Comment by Iceber** [comment, marked as answer]
If you're not using the memory or sqlite storage layer, then you deploy the apiserver.


the `binding-apiserver` is the equivalent of `apiserver` + `clustersynchro manager`, integrating the two functions in one application.
Of course `binding-apiserver` is not a very good name and may be changed in the future!
